Introduction_to_RMarkdown.Rmd

```diff
@@ -0,0 +1,195 @@
+# Tutorial on R Markdown
```

Comment:
The .Rmd filename can only contain lowercase letters. (Otherwise the filenames do not sort nicely on the repo home page.) please correct

Reply:
Hi， I am wondering how to resubmit a new .rmd file?

Reply:
You might be able to change the name of your file straight from your branch? 

This is from the submission instructions:  If you are asked to make changes, simply make the changes on your local branch, save, commit, and push to GitHub. The new commits will be added to your pull request; you do not need to, and should not, create a new pull request. (If, based on the circumstances, it does make sense to close the pull request and start a new one, we will tell you.)

Reply:
Thank you for your instruction and I pushed a new file this morning. Can you have a check whether you can see it and whether can it be merged successfully this time?

Reply:
@jtr13 i attempted to merge this since Annie fixed the issue but received an Run failed message from Github, is there anything that I should do?